text/1105-api-evolution.md

```diff
@@ -22,6 +22,10 @@ The RFC covers only API issues; other issues related to language features,
 lints, type inference, command line arguments, Cargo, and so on are considered
 out of scope.
 
+The stability promise specifically does *not* apply to unstable features,
+even if they are accidentally usable on the Stable release channel
+under certain conditions such as because of bugs in the compiler.
```

Comment:
Hmm; Maybe say 1-2 words about who decides this? (I assume the responsible team does?)

Reply:
This PR is intended to be the decision?

Reply:
What I mean is: if we in the future find new "bugs" wrt. unstable; who decides if it was really a bug, or working as intended? I guess the criteria for bug could be / is that something became stable without going through a stabilization FCP?

Reply:
In the standard library at least I think there’s no ambiguity since every public item is required to have either `#[stable]` or `#[unstable]`. Though you’re right that it’s may not that clear cut for language features.

Reply:
Small note re. standard library: trait impls are insta-stable regardless of `#[unstable]` afaik and macros do not respect stability attributes either.